locations/spiders/burger_king_ru.py

```
@@ -22,17 +21,20 @@ async def start(self) -> AsyncIterator[JsonRequest]:
         yield JsonRequest(url=self.api_url, cookies={"spid": "", "spsc": ""})
 
     def parse(self, response: Response, **kwargs: Any) -> Any:
-        for location in json.loads(response.xpath("//pre/text()").get())["response"]:
+        for location in response.json()["response"]:
             item = DictParser.parse(location)
             item["street_address"] = item.pop("name", None)
             item["city"] = location["city"]["city_name"]
+            item["phone"] = None  # A single national number, varying only by extension
             try:
                 item["opening_hours"] = self.parse_opening_hours(location.get("timetable", []))
             except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)